core/src/main/scala/org/apache/spark/deploy/raydp/RayAppMaster.scala

```diff
@@ -262,6 +277,14 @@ class RayAppMaster(host: String,
       val classPathEntries = appInfo.desc.command.classPathEntries.mkString(";")
       RayExecutorUtils.setUpExecutor(handlerOpt.get, appId, driverUrl, cores, classPathEntries)
     }
+
+    private def getNextBundleIndex: Int = {
+      if (placementGroup != null && bundleIndexes.nonEmpty) {
+        val previous = currentBundleIndex
+        currentBundleIndex = (currentBundleIndex + 1) % bundleIndexes.size
+        previous
+      } else -1
```

Comment:
Let's follow Spark's code style. Unless you can write if else on the same line, otherwise always use {}. This should looks like 
```
} else {
  -1
}
```